v2/robotmk/session.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
This feature will perhaps never make it into python, better just reorder the class definitions and remove this line.